### PR TITLE
Adding Template for Slides and Updating Readme

### PR DESCRIPTION
**Pull request by swzCuroverse**
Adding the template we used for the CZI EOSS talk and updated the readme to point to it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,10 @@ If used under CC-BY-SA we'll consider http://www.commonwl.org/ or [10.6084/m9.fi
 
 * Modern **website** or vector editing (e.g. [Inkscape](https://inkscape.org/en/) or Adobe InDesign)?: The _scalable_ [CWL-Logo-nofonts.svg](CWL-Logo-nofonts.svg) should probably work (you may want to scale it with CSS or `width` tag)
   * Prefer letters? Try original export [CWL-Logo.svg](CWL-Logo.svg) with the font [SourceSansPro-Light](https://github.com/adobe-fonts/source-sans-pro)
-* Adding to **Powerpoint**? [CWL-Logo-VGA.png](CWL-Logo-VGA.png) for slides, or [CWL-Logo-4k.png](CWL-Logo-4k.png) for print
+* Adding to **Powerpoint**? [CWL-Logo-VGA.png](CWL-Logo-VGA.png) for slides, or [CWL-Logo-4k.png](CWL-Logo-4k.png) for print 
+* Need **Powerpoint** Template? [CWL-Slides-Template.pptx](CWL-Slides-Template.pptx) can be used for slides
 * Making a **poster** or print material in editor that can't import SVG (really?) [CWL-Logo-4k.png](CWL-Logo-4k.png)
-* Old-school lo-res website **banner**? [CWL-Logo-Header.png](CWL-Logo-Header.png)
+* Old-school lo-res website **banner**? [](CWL-Logo-Header.png)
```
Comment (mr-c) [inline README.md:26]: Hello @swzCuroverse ; why was this deleted?

Comment (swzCuroverse) [reply README.md:26]: No idea - I think it was a typo?  Let me fix it and repull.
